### PR TITLE
add PDB/updateStrategy/priorityClass

### PR DESCRIPTION
**Pull request by cupnes**
- add pod disruption budget
- add updateStrategy for lvmd and topolvm-node
- add priorityClass

Signed-off-by: Yuma Ogami <yuma-ogami@cybozu.co.jp>

### CODE REVIEW

**Review by toshipp — APPROVED** [review]

**Review by toshipp — APPROVED** [review]

File: charts/topolvm/templates/controller/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{{- if or .Values.controller.podDisruptionBudget.enabled }}
```
Comment (llamerada-jp) [inline charts/topolvm/templates/controller/poddisruptionbudget.yaml:1]:
```suggestion
{{- if .Values.controller.podDisruptionBudget.enabled }}
```

File: charts/topolvm/templates/scheduler/poddisruptionbudget.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{{- if or .Values.scheduler.podDisruptionBudget.enabled }}
```
Comment (llamerada-jp) [inline charts/topolvm/templates/scheduler/poddisruptionbudget.yaml:1]:
```suggestion
{{- if .Values.scheduler.podDisruptionBudget.enabled }}
```

File: charts/topolvm/templates/priorityclass.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{{- if or .Values.priorityClass.enabled }}
```
Comment (llamerada-jp) [inline charts/topolvm/templates/priorityclass.yaml:1]:
```suggestion
{{- if .Values.priorityClass.enabled }}
```

File: charts/topolvm/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,11 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+## [3.1.0] - 2021-09-30
+
+### Changed
+- add PDB/updateStrategy/priorityClass (#370)
```
Comment (llamerada-jp) [inline charts/topolvm/CHANGELOG.md:13]: Has this PR not been released yet?

**Review by llamerada-jp — CHANGES_REQUESTED** [review]

**Review by toshipp — APPROVED** [review]

**Review by satoru-takeuchi — APPROVED** [review]

**Review by llamerada-jp — APPROVED** [review]

**Pull request merged**